Packs/Armis/Integrations/Armis/Armis.py

```diff
@@ -0,0 +1,669 @@
+                'deviceIds'
+            ], removeNull=True)
+        )
+    return 'No results found'
+
+
+def search_devices_command(client: Client, args):
+    """
+    armis-search-devices command: Returns results for searching Devices by common parameters
+    Args:
+        client (Client): An Armis client object
+        args (dict): A dict object containing the arguments for this command
+    """
+    risk_level = demisto.args().get('risk_level')
+    if risk_level is not None:
+        risk_level = argToList(risk_level)
+
+    device_type = demisto.args().get('device_type')
+    if device_type is not None:
+        device_type = argToList(device_type)
+
+    name = args.get('name')
+    device_id = args.get('device_id')
+    mac_address = args.get('mac_address')
+    ip_address = args.get('ip_address')
+    time_frame = args.get('time_frame')
+    max_results = int(args.get('max_results', 50))
+
+    response = client.search_devices(name,
+                                     device_id,
+                                     mac_address,
+                                     risk_level,
+                                     ip_address,
+                                     device_type,
+                                     time_frame,
+                                     max_results=max_results)
+    results = response.get('results')
+    if results:
+        return CommandResults(
+            outputs_prefix='Armis.SearchDevices',
+            outputs_key_field='deviceId',
+            outputs=results,
+            raw_response=response,
+            readable_output=tableToMarkdown('Devices', results, headers=[
+                'riskLevel',
+                'name',
+                'dataSources',
+                'category',
+                'type',
+                'model',
+                'manufacturer',
+                'macAddress',
+                'ipAddress',
+                'site',
+                'tags',
+                'operatingSystem',
+                'operatingSystemVersion',
+                'sensor',
+                'user',
+                'accessSwitch',
+                'firstSeen',
+                'id',
+                'ipv6',
+                'lastSeen',
+                'visibility',
+            ], removeNull=True)
+        )
+    return 'No devices found'
+
+
+def search_devices_by_aql_command(client: Client, args):
+    """
+    armis-search-devices-by-aql command: Returns results for searching Devices using a free AQL string
+    Args:
+        client (Client): An Armis client object
+        args (dict): A dict object containing the arguments for this command
+    """
+    aql_string = args.get('aql_string')
+    max_results = int(args.get('max_results', 50))
+
+    response = client.free_string_search_devices(aql_string, max_results=max_results)
+    results = response.get('results')
+    if results:
+        return CommandResults(
+            outputs_prefix='Armis.SearchDevicesByAql',

```

Comment:
```suggestion
            outputs_prefix='Armis.Device',

```

So we want all same objects to be present under the same context path.
This means that both commands that returned devices will return them to the same path and then all devices will be present there. 
This is also true for alerts (I didn't comment there but please fix there too). If it's not clear lets have a quick call